src/offset/utc.rs

```diff
@@ -109,6 +101,19 @@ impl Utc {
         let now = js_sys::Date::new_0();
         DateTime::<Utc>::from(now)
     }
+
+    /// Returns a `DateTime` which corresponds to the current date and time.
+    #[cfg(all(target_arch = "wasm32", feature = "wasmbind", target_os = "emscripten"))]
+    #[must_use]
+    pub fn now() -> DateTime<Utc> {
+        use emscripten_functions::emscripten::run_script_string;
+        // In order to handle number than i32, string is used to store the number
+        let now = run_script_string("Date.now().toString()")
+            .expect("Date.now() failed")
+            .parse::<i64>()
+            .unwrap();
```

Comment:
Better to have something different than `unwrap` here. Maybe use `map` or `and_then`?